lib/teracy-dev/location/git_synch.rb

```diff
@@ -92,6 +110,8 @@ def sync(location, sync_existing)
           Dir.chdir(lookup_path) do
             @logger.info("cd #{lookup_path} && git clone #{git_remote_url} #{dir}")
             system("git clone #{git_remote_url} #{dir}")
+
```

Comment:
If you already have `GITHUB_USERNAME and GITHUB_PASSWORD` and this clone is still fail, it will trigger `abort_process_if_fail_to_clone` and will attempting to use `GITHUB_USERNAME and GITHUB_PASSWORD`